src/Application/Console/Definitions/DefaultDefinition.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Application\Console\Definitions;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * Minimum definition for each of our commands.
```

Comment:
This comment is not saying the same as the class name. Is it the default or the required definition for each command?